google-cloud-firestore-admin/src/test/java/com/google/cloud/firestore/v1/FirestoreAdminClientTest.java

```diff
@@ -75,19 +75,17 @@
 @Generated("by gapic-generator-java")
 public class FirestoreAdminClientTest {
   private static MockFirestoreAdmin mockFirestoreAdmin;
-  private static MockLocations mockLocations;
   private static MockServiceHelper mockServiceHelper;
   private LocalChannelProvider channelProvider;
   private FirestoreAdminClient client;
 
   @BeforeClass
   public static void startStaticServer() {
     mockFirestoreAdmin = new MockFirestoreAdmin();
-    mockLocations = new MockLocations();
     mockServiceHelper =
         new MockServiceHelper(
             UUID.randomUUID().toString(),
-            Arrays.<MockGrpcService>asList(mockFirestoreAdmin, mockLocations));
+            Arrays.<MockGrpcService>asList(mockFirestoreAdmin));
```

Comment:
`@Generated("by gapic-generator-java")`. It shouldn't have to be done manually.

Reply:
The path of the FirestoreAdminClientTest.java is

`google-cloud-firestore-admin/src/test/java/com/google/cloud/firestore/v1/FirestoreAdminClientTest.java`


OwlBot's configuration file https://github.com/googleapis/java-firestore/blob/main/.github/.OwlBot.yaml

Reply:
@dconeybe Let's merge https://github.com/googleapis/java-firestore/pull/1032 first.